src/utilcode/stresslog.cpp

```diff
@@ -314,6 +314,7 @@ ThreadStressLog* StressLog::CreateThreadStressLog() {
         // we fail.
         ClrFlsSetValue(theLog.TLSslot, NULL);
     }
+#pragma warning(suppress: 4101)
```

Comment:
I think you need to disable suppression then.

Reply:
@sandreenko according to https://msdn.microsoft.com/en-us/library/2c8f766e.aspx `suppress` will suppress the warning only once, hence not needing to restore the warning

Reply:
Good, thanks.